src/server.cpp

```diff
@@ -3097,15 +3097,16 @@ std::string Server::getStatusString()
 	std::ostringstream os(std::ios_base::binary);
 	os << "# Server: ";
```

Comment:
`Server: version: xxxx` looks strange. How about combing them because the user expects a server status message anyway? -> `# Server version: xxxx | uptime: xxxx | ...`